spec/fixtures/cleanup-foreman-user-groups/foreman-answers-after.yaml

```diff
@@ -0,0 +1,2 @@
+foreman:
+  user_groups: []
```

Comment:
```suggestion
foreman:
```

Should we add a 2nd piece of logic to the migration to remove the key entirely if the value ends up `[]`?

Reply:
I prefer not to try to predict that situation and just do the targeted cleanup this is aimed at.

Reply:
Actually, I suppose it would come back anyway after the installer run, because `[]` is the default in puppet-foreman